probe/endpoint/reporter.go

```diff
@@ -64,12 +65,16 @@ func NewReporter(hostID, hostName string, includeProcesses bool, useConntrack bo
 			log.Printf("Failed to start natMapper: %v", err)
 		}
 	}
+
+	revRes := NewReverseResolver(rAddrCacheLen)
```

Comment:
Also, name consistency with conntracker and natmapper would lead me to suggest revresolver.